exploration/0005-open-close-expressions.md

```diff
@@ -0,0 +1,152 @@
+# Open/Close Expressions
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@eemeli</dd>
+		<dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2023-09-05</dd>
+		<dt>Pull Request</dt>
+		<dd>#000</dd>
+	</dl>
+</details>
+
+## Objective
+
+_What is this proposal trying to achieve?_
+
+Describe the use cases and requirements for _expressions_
+that enclose parts of a pattern
+and develop a design that satisfies these needs.
+
+## Background
+
+_What context is helpful to understand this proposal?_
+
+## Use-Cases
+
+- Representing markup, such as HTML, as expressions in a pattern
+  rather than as plain character sequences embedded in the _text_
+  of a pattern. This also allows parts of a markup sequence to be
+  programmable.
+
+**_Non-markup use cases to consider (which are not addressed by the design)_**
+
+- Represent templating language constructs so that they can pass through
+  the translation process and are not exposed to translation. For example,
+  Mustache templates.
```

Comment:
@aphillips I'm pretty sure that this is covered by the current "proposed design". Or is there some aspect of this that I'm missing? A concrete example would help.

```suggestion
- Represent templating language constructs so that they can pass through
  the translation process and are not exposed to translation.
  For example, Mustache templates.

**_Non-markup use cases to consider (which are not addressed by the design)_**
```

Reply:
That certainly works for me. Getting an example or two included in the doc would help explain what this is about.